.github/workflows/lint.yml

```diff
@@ -120,6 +120,17 @@ jobs:
         run: |
           find doc/ -name '*.py' | xargs flake8
           find tests/ -name '*.py' | xargs flake8
+        
+      - name: Check integration test IDs
```

Comment:
Just wondering: as this is integration tests specific, wouldn't it be better to have that check in integration-tests.yml as a dependency of rpmbuild job?

Reply:
Good point and I agree, it should be in `integration-tests.yml`. I moved it and, since there is `tmt` available, use `tmt test id --dry` instead of find + grep. Could you check again? @mwperina